topic7/src/main/java/controllers/CourseController.java

```diff
@@ -0,0 +1,51 @@
+package controllers;
+
+import models.Course;
+import models.CourseDao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class CourseController {
+
+	@Autowired
+	private CourseDao _CourseDao;
+
+	@RequestMapping("/create")
+	@ResponseBody
+	public String create(String name) {
+		try {

```

Comment:
It's a good practice use try-catch to avoid to un-catch possible errors in your code, but keep in mind that if you are implementing a web service it always need to return a message to FE indicating the result of the operation, in that case an issue (it is possible using an ExceptionHandler)